Gruntfile.js

```diff
@@ -58,12 +58,18 @@ module.exports = function(grunt) {
 							slowTag = '--grep @slow';
 						} else if (tag === 'unstable') {
 							slowTag = '--grep @unstable';
+						} else if (tag === 'network') {
+							slowTag = '--grep @network';
+						} else if (tag === 'propagation') {
+							slowTag = '--grep @propagation';
+						} else if (tag === 'stress') {
+							slowTag = '--grep @stress';
 						} else {
 							grunt.fail.fatal(
 								'The specified tag is not supported.\n\nExample: `grunt mocha:<tag>:<suite>:[section]` or `npm test -- mocha:<tag>:<suite>:[section]`\n\n- Where tag can be one of slow | unstable | untagged | extensive (required)\n- Where suite can be one of unit | functional | integration (required)\n- Where section can be one of get | post | ws | system (optional)'
 							);
 						}
-						return `./node_modules/.bin/_mocha --bail test/integration/index.js ${slowTag}`;
+						return `./node_modules/.bin/_mocha --bail test/integration/integration.js ${slowTag}`;
```

Comment:
I don't know exactly why this change was required. But having an `index.js` file under directory to reflect entry point of integration tests makes more sense to me.

Reply:
I wanted to follow the same pattern as in functional. It doesn't change anything...